src/Spice86.Core/Emulator/CPU/CfgCpu/ControlFlowGraph/ICfgNode.cs

```diff
@@ -50,4 +51,10 @@ public interface ICfgNode {
     /// <param name="helper">InstructionExecutionHelper instance providing access to the outside</param>
     public void Execute(InstructionExecutionHelper helper);
 
+    /// <summary>
+    /// Renders the node as assembly code
+    /// </summary>
+    /// <param name="helper">helper for rendering</param>
+    /// <returns></returns>
+    public string ToAssemblyString(InstructionRendererHelper helper);
```

Comment:
A better name is instructionRenderer

Reply:
I am going to rewrite this completely but I will keep that in mind :)